### PR TITLE
allow write_secuTrial() to write xpt files

### LINKED ISSUE #57: Allow for xpt files to be created.
**Is your feature request related to a problem? Please describe.**
When writing to xpt the variable names are truncated.

**Describe the solution you'd like**
Allow for xpt files to be created.

**Additional context**
Currently the lines of code in write_secutrial.R regarding xpt files are commented. Also the tests regarding xpt files have been deactivated for now.

### PR DESCRIPTION
**Pull request by PatrickRWright**
Looks like the new `haven 2.2.0` resolved this. I just reintroduced the commented code and the tests are now passing. I also ran a manual test with local data and it seems fine.

Unfortunately, the column names in xpt files are truncated but I think our hands are tied here since that likely happens to conform with SAS standards.

closes #57

### CODE REVIEW

**Comment by PatrickRWright** [comment]
Oh yes. Good catch.

File: R/write_secuTrial.R [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+#'              using the haven package.
 #' @name write_secuTrial
 #' @param object secuTrialdata object
 #' @param path directory where the files should be saved
-# @param format format in which to save the export (one of "dta", "sas", "sav", "xpt")
-#' @param format format in which to save the export (one of "dta", "sas", "sav")
-#' @details Due to variable naming limitations in other packages, date variables are appended with _d (rather than _date), datetime/POSIX variables are appended with _dt (rather than _datetime) and factors with _f (rather than _factor).
+#' @param format format in which to save the export (one of "dta", "sas", "sav", "xpt")
+#' @details Due to variable naming limitations in other packages, date variables are
+#'          appended with _d (rather than _date), datetime/POSIX variables are appended
```
Comment (aghaynes) [inline R/write_secuTrial.R]: do we need other notes about name truncation?

Comment (PatrickRWright) [reply R/write_secuTrial.R]: The thing is that I don't really know which rules are applied by haven.

Comment (aghaynes) [reply R/write_secuTrial.R]: sounds fair enough to me

Comment (PatrickRWright) [reply R/write_secuTrial.R]: Ok, I'll make the adjustments.

**Comment by PatrickRWright** [comment]
Thanks @aghaynes 
Can you merge this when travis is done?

File: R/write_secuTrial.R [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-#' @description Convert the export prepared in R and export it SPSS, Stata or SAS using the haven package.
+#' @description Convert the export prepared in R and export it to
+#'              SPSS (sav), Stata (dta) or SAS (sas, xpt version 8)
+#'              using the haven package.
 #' @name write_secuTrial
 #' @param object secuTrialdata object
 #' @param path directory where the files should be saved
-# @param format format in which to save the export (one of "dta", "sas", "sav", "xpt")
-#' @param format format in which to save the export (one of "dta", "sas", "sav")
-#' @details Due to variable naming limitations in other packages, date variables are appended with _d (rather than _date), datetime/POSIX variables are appended with _dt (rather than _datetime) and factors with _f (rather than _factor).
```
Comment (aghaynes) [inline R/write_secuTrial.R]: I would actually retain this as it's secuTrialR not haven... or did you remove that functionality? haven might do other stuff though...

Comment (PatrickRWright) [reply R/write_secuTrial.R]: Ah, I didn't realize we were doing that. I will put it back in.

Comment (aghaynes) [reply R/write_secuTrial.R]: Thanks

**Pull request merged**